git-fixup

```diff
@@ -176,6 +182,9 @@ while test $# -gt 0; do
             fi
             base="$1"
             ;;
+        -a|--all)
```

Comment:
Could you also add this flag into the tab completion files?

Reply:
done